schemas/alpha/azure.yaml.json

```diff
@@ -1047,6 +1071,111 @@
             "required": [
                 "deployment"
             ]
+        },
+        "deploymentStacksConfig": {
+            "type": "object",
+            "title": "The deployment stack configuration used for the project.",
+            "additionalProperties": false,
+            "oneOf": [
```

Comment:
What does this end up meaning?  Does it mean if you're going to have a `deploymentStacksConfig` object then it must have at least one property (i.e. disallowing the empty object as the value)?

Reply:
Yes, if the user bothers to add the `deploymentStacks` property, then enforce that they set at least one of the top level sections otherwise they can just omit the whole thing and use the internal defaults.